.travis.yml

```diff
@@ -64,7 +67,9 @@ install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
   - "npm update"
-  - "npm install --silent clean-css@3.4.19 requirejs"
+  - "npm install --silent clean-css@${CLEANCSS} requirejs"
+  # If clean-css v. 4+, we need to install clean-css-cli. It will fail if we use v. 3.
+  - "npm install --silent clean-css-cli@${CLEANCSS} || true"
```

Comment:
here, little trick... Version 4 of cleancss has been splitted in 2 packages: `clean-css` and `clean-css-cli`. So, if we are testing the fourth version, we need the latest.
On the other hand, if we are testing the version 3, the CLI package doesn't exist, so the install will fail. But it's OK!